### ISSUE TITLE
add human results as a secondary plot next to each interactive plot

### ISSUE THREAD

**Issue by wkvong** [alexatartaglini/alcove#14]

**Comment by alexatartaglini** [comment]
<img width="979" alt="Screen Shot 2020-06-08 at 5 23 14 PM" src="https://user-images.githubusercontent.com/48462071/84090032-d5d27f80-a9ad-11ea-8899-e0d192b6c7e2.png">
Done! Does this look good?

**Comment by wkvong** [comment]
yeah, this looks really awesome! its really cool to be able to compare them side-by-side like this. do you have any idea why the model's predictions seem to plateau below 0.9 rather than getting to 1 by any chance?

**Comment by alexatartaglini** [comment]
<img width="471" alt="Screen Shot 2020-06-08 at 5 41 37 PM" src="https://user-images.githubusercontent.com/48462071/84090645-9c027880-a9af-11ea-8ccb-eb033261e162.png">
Here's a screenshot of part of the CSV. The calculation for probability correct isn't something I've changed/written, so I'm not entirely sure why it seems to cap at 0.8; is it supposed to be 1-loss?

**Comment by wkvong** [comment]
I'm not sure to be honest, do you get the same pattern of results with the other loss functions in the abstract + ALCOVE setting? In the original figure from Kruschke's paper the curves don't get to 100%, but p(correct) for Type 1 is above 95%, which is higher than what we're seeing here. I can look into it myself too and see if I can explain what is happening here!
